server/cluster/cluster.go

```diff
@@ -2511,6 +2511,16 @@ func (c *RaftCluster) GetMinResolvedTS() uint64 {
 	return c.minResolvedTS
 }
 
+// GetStoreMinResolvedTS returns the min resolved ts of the store.
+func (c *RaftCluster) GetStoreMinResolvedTS(storeID uint64) uint64 {
+	c.RLock()
+	defer c.RUnlock()
+	if !c.isInitialized() {
```

Comment:
Do we also need to check `IsAvailableForMinResolvedTS` here?

Reply:
added.